db/upgrade.php

```diff
@@ -817,7 +818,7 @@ function xmldb_zoom_upgrade($oldversion) {
         $table = new xmldb_table('zoom');
 
         // Define and conditionally add field option_auto_recording.
-        $field = new xmldb_field('option_auto_recording', XMLDB_TYPE_CHAR, '5', null, null, null, null, 'show_media');
+        $field = new xmldb_field('option_auto_recording', XMLDB_TYPE_CHAR, '5', null, XMLDB_NOTNULL, null, 'none', 'show_media');
```

Comment:
While the current change would fix the inconsistency, any users who have not yet upgraded should be given the new-correct field definition so that they don't end up with rows that have the wrong defaults.
```suggestion
        $field = new xmldb_field('option_auto_recording', XMLDB_TYPE_CHAR, '5', null, null, null, null, 'show_media');
```